src/components/navigation-dropdown/navigation-dropdown.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ControlSelect from '@mapbox/react-control-select';
+// import { routeToPrefixed } from '@mapbox/batfish/modules/route-to';
+
+class NavigationDropdown extends React.Component {
+  render() {
+    const { props } = this;
+    const optionsArray = props.dropdownOptions.map(option => {
+      return {
+        label: option.title,
+        value: option.path
+      };
+    });
+    const currentOption = optionsArray.filter(option => {
+      return props.currentPath === option.value;
+    });
```

Comment:
`currentOption` is actually the first item in the array, so I might grab that here so the variable name is more clear

Reply:
```js
const currentOption = optionsArray.filter(option => {
  return props.currentPath === option.value;
})[0];
```

That `[0]` at the end means `currentOption` will be the current option instead of an array with one item that is the current option.

Reply:
You *could* use `find` instead of `filter`, except find isn't supported on IE11 🤷‍♂️

Reply:
Oh I understand. Thanks!